torch/testing/_internal/common_methods_invocations.py

```diff
@@ -11624,6 +11645,9 @@ def ref_pairwise_distance(input1, input2):
                        # Reference: https://github.com/pytorch/pytorch/pull/52551#issuecomment-782596181
                        DecorateInfo(unittest.skip("Skipped!"), 'TestUnaryUfuncs', 'test_reference_numerics_hard',
                                     dtypes=[torch.bfloat16]),
+                       # Forward Mode AD support not added yet for square
+                       DecorateInfo(unittest.skip("Skipped!"), 'TestGradients', 'test_inplace_forward_mode_AD'),
+                       DecorateInfo(unittest.skip("Skipped!"), 'TestGradients', 'test_forward_mode_AD'),
```

Comment:
relevant change

Reply:
What is the error here? You've included `result: auto_element_wise` in the `derivatives.yaml` so forward AD should be working.